lib/digitizer_block_impl.cc

```diff
@@ -967,9 +967,8 @@ namespace gr {
            d_app_buffer.notify_data_ready(digitizer_block_errc::Watchdog);
 
          }
-         boost::chrono::duration<float> poll_duration = boost::chrono::high_resolution_clock::now() - poll_start;
-
-         boost::this_thread::sleep_for(poll_rate - poll_duration);
+         std::chrono::duration<float> poll_duration = std::chrono::high_resolution_clock::now() - poll_start;
+         std::this_thread::sleep_for(poll_rate - poll_duration);
```

Comment:
maybe rename `poll_rate` to `polling_period`?

Reply:
renamed "poll_duration" to "poll_duration_correction" and "poll_rate" to
"poll_duration" and added a comment to make clear what is going on .. fine like this ?